### PR TITLE
chore: export baggage

### PR DESCRIPTION
**Pull request by dyladan**
Baggage types were not exported

### CODE REVIEW

File: src/baggage/index.ts [hunk 1]
```diff
@@ -14,66 +14,6 @@
...
-  }
-
-  return {
-    __TYPE__: baggageEntryMetadataSymbol,
-    toString() {
-      return str;
-    },
-  };
-}
+export * from './context-helpers';
```
Comment (vmarchaud) [inline src/baggage/index.ts:17]: we shouldn't export context helpers or `createBaggage` a the top level, so we should remove this line and line 18 should be `export { baggageEntryMetadataFromString } from './types';`

Comment (dyladan) [reply src/baggage/index.ts:17]: Why would we not export them? Context helpers just get and set baggage on the context, an operation people may reasonably want to perform. How do you access baggage without them?

Comment (vmarchaud) [reply src/baggage/index.ts:17]: Because they are exposed on the `propagation` namespace, that was the point of #55

Comment (dyladan) [reply src/baggage/index.ts:17]: Latest commit look ok then?

Comment (obecny) [inline src/baggage/index.ts:17]: hmm this line shouldn't be exported ?

**Review by obecny — APPROVED** [review]
I think it looks fine now, just wait for confirmation yet from @vmarchaud yet

**Review by vmarchaud — APPROVED** [review]

File: src/baggage/index.ts (hunk 1 above)
Comment (vmarchaud) [reply src/baggage/index.ts:17]: :+1:

Comment (dyladan) [reply src/baggage/index.ts:17]: Maybe we should remove the index.ts file entirely and reference things directly to remove confusion? Or we could move the context helpers into the `internal` folder to show they should not be exported.

**Pull request merged**